android/app/src/main/java/com/microsoft/codepush/react/CodePushUtils.java

```diff
@@ -265,6 +271,108 @@ public static String tryGetString(ReadableMap map, String key) {
         }
     }
 
+    public static void unzipFile(File zipFile, String destination) throws IOException {
+        FileInputStream fis = null;
+        BufferedInputStream bis = null;
```

Comment:
Point taken about the naming. And no, local variables are not auto-initialized to null in Java. It only does that for class variables.